src/types.rs

```diff
@@ -534,6 +604,24 @@ impl TxEventsResponse {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SignedMsgResponse {
+    pub signed_msg_order_results: Vec<SignedMsgOrderResult>,
```

Comment:
```suggestion
    #[serde(rename = "results")
    pub signed_msg_order_results: Vec<SignedMsgOrderResult>,
```
or just rename the field